ssh-proxy/orb.yml

```diff
@@ -63,7 +63,7 @@ commands:
     steps:
       - run:
           name: Release all sshuttle sessions, which will kill forked ssh sessions as well
-          command: sudo sh -c "ps a | grep -- 'sshuttle' | grep -v 'grep' | awk '{print $2}' | xargs kill"
+          command: sudo sh -c "ps au | grep -- 'sshuttle' | grep -v 'grep' | awk '{print $2}' | xargs kill"
```

Comment:
`sudo pkill sshuttle`?

Reply:
Yeah, that's much cleaner, I'll change that